Tasks/Common/webdeployment-common/WebConfigTemplates/node

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is the file in the common folder. If yes, then consider the logic of generating web.config file to a separate file in the common folder.